bcachefs/bcachefs.c

```diff
@@ -412,6 +492,8 @@ int Bcachefs_close(Bcachefs *this)
 
 int Bcachefs_iter(const Bcachefs *this, Bcachefs_iterator *iter, enum btree_id type)
 {
+    *iter = (Bcachefs_iterator){0};
+
```

Comment:
Link to [previous discussion](https://github.com/mila-iqia/bcachefs/pull/12#discussion_r732019157)